scripts/install_xcube-datastore.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/env bash
```

Comment:
Can we pls rename this file to just `install-xcube.sh`, if there is no special reason for `-datastore` sufix

Reply:
done